### PR TITLE
Update to 2022-11-03_20-12-06Z

### PR DESCRIPTION
**Pull request by tratnayake**
* Security patch to `openssl` library used by the microsat. (Patches CVE's 2022-3602 and 2022-3358). See: https://launchpad.net/ubuntu/+source/openssl/3.0.5-2ubuntu2

R: @mi-lee

### CODE REVIEW

**Comment by mattcobb** [comment]
I think it is missing this step:  Please delete the charts/lightstepsatellite/README.md and regenerate it with the [helm-docs](https://github.com/norwoodj/helm-docs) command.

File: charts/lightstepsatellite/README.md [hunk 1]
```diff
@@ -100,4 +100,4 @@ Lightstep microsatellite to collect telemetry data.
 | tolerations | list | `[]` |  |
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```
Comment (mi-lee) [inline charts/lightstepsatellite/README.md:103]: why this change?

Comment (mi-lee) [reply charts/lightstepsatellite/README.md:103]: oh we need to update it here: https://github.com/lightstep/lightstep-microsatellite-helm-chart/blob/main/.github/helm-docs.sh#L4

**Comment by flamein** [comment]
https://github.com/lightstep/lightstep-microsatellite-helm-chart/pull/31 supersedes this one
